lib/lz4.h

```diff
@@ -391,7 +393,7 @@ typedef struct {
  *        this definition is not API/ABI safe,
  *        it may change in a future version !
  */
-#define LZ4_STREAMSIZE_U64 ((1 << (LZ4_MEMORY_USAGE-3)) + 4)
+#define LZ4_STREAMSIZE_U64 ((1 << (LZ4_MEMORY_USAGE-3)) + 5)
```

Comment:
One way to avoid increasing the size of `LZ4_stream_t_internal`
is to take the slot currently occupied by `bufferStart`.
This pointer is no longer useful.
Removing it is likely going to trigger some side-effects in some obsolete functions,
but I believe it's fine, these functions have been obsolete for many years by now.
For discussion.

Reply:
If we feel we can remove those functions, then yeah, we could do that. I looked at them though and couldn't think of an easy way to remove `bufferStart` and retain their functionality.

Reply:
I don't think it's possible to retain their functionality.
I suspect no one uses these deprecated functions.